docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+| `Licensing - File`          | `PackageLicenseFile`         | `license type="file"`       | File for custom license that isn't supported by license expressions                                               |
+| `Project URL`               | `PackageProjectUrl`          | `projectUrl`                | A URL for the project homepage.                                                                                   |
+| `Icon File`                 | `PackageIcon`                | `icon`                      | An image to use as the icon for the package.                                                                      |
+| `Repository URL`            | `RepositoryUrl`              | `repository url=""`         | URL to the repository from which the package was built.                                                           |
+| `Repository type`           | `RespositoryType`            | `repository type=""`        | Type of repository the repository URL is pointing to (i.e. git)                                                   |
+| `Tags`                      | `PackageTags`                | `tags`                      | A space-delimited list of tags and keywords that describe the package. Tags are used when searching for packages. |
+| `Release notes`             | `PackageReleaseNotes`        | `releaseNotes`              | A description of the changes made in this release of the package.                                                 |
+
+### Package ID
+
+If you're publishing a completely new package:
+
+✔️ DO choose a package ID that is unique and clearly differentiated from existing packages on NuGet.org.
+> You can make sure a package ID is unique and differntiable by searching for the ID on NuGet.org or checking if the following link exists: https://www.nuget.org/packages/<package name\>.
+
+✔️ CONSIDER choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation#id-prefix-reservation-criteria).
+> Reserving the prefix ID for your package will let you get the verified check mark:
+> ![image](media/Verified-check-mark.png)
+> Check out the [Package ID prefix reservation docs](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation) to learn more.
+
+### Package Version
+
+✔️ CONSIDER using [SemVer 2.0.0](https://semver.org/) to version your NuGet package.
+> Essentially, this means using the Major.Minor.Patch[-prerelease] format.
+
+✔️ DO include a pre-release suffix when releasing a non-stable package.
+
+See the [.NET library versioning guide](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/versioning) for more advanced guidance including aseembly versioning.
+
+### Authors
+
+✔️ CONSIDER including the current NuGet.org owners for consistency across NuGet clients.
```

Comment:
I don't think I agree with this one. Authors is more generally used for a pretty human or company name where as username is often abbreviated and lowercase.
![image](https://user-images.githubusercontent.com/94054/93631020-c77a0380-f99f-11ea-943c-a3bdeede19f8.png)

Reply:
That's true, but currently we require consistency between Owner, Author, and prefix for prefix reservations. So it'd be weird that that's not a "best practice." It's a bit of a band-aid for not showing Owners in the PMUI Browse tab, and I agree that it's kind of weird.
@anangaur @JonDouglas thoughts on this one? Personally, I was a big conflicted.

Reply:
Author is used for 2 purposes today:
1. To use a better nam, may be trademarked one. E.g. `My Foo Company Limited` corresponding to owner = `myfoo`
2. To credit the actual author(s) that's different from owner. e.g. `Microsoft and Collaborators` or `<Person name>`

IMO we could say make it the same as the owner with a pretty name. Thoughts?

Reply:
You mean like

Authors: NuGet, Chris Gill

?

Reply:
So have both Owners + Pretty Name?

Reply:
Using it as a band-aid for showing Owners in the PMUI is not great. First, people can lie (putting `Microsoft` in the authors field), so any emphasis on authors == owners or owners == authors is a bad signal to both package authors and consumers. Second, owners can change over time. For example, Newtonsoft.Json used to only be owned by `jamesnk` but not it's owned by a new organization and .NET Foundation.

The author should be use _authored_ the package. It's like a book publisher vs. book author (in my mind). nuget.org is the book store in that analogy.

Reply:
@anangaur, I think a trademarked and/or pretty name for the author is good. So for Microsoft packages it's Microsoft but for Newtonsoft.Json it's James Newton-King.

Reply:
I agree with @joelverhagen. 

This recommendation falls apart quickly when the author is an individual and not a company.